examples/playwright/src/theia-preference-view.ts

```diff
@@ -33,6 +37,19 @@ export const PreferenceIds = {
 };
 
 export const DefaultPreferences = {
+    Editor: {
+        AutoSave: {
+            On: 'on',
+            Off: 'off'
+        },
```

Comment:
@planger I'm a bit confused as to how does it works, the preference `id` and `values` do not match our latest schema:

https://github.com/eclipse-theia/theia/blob/acd6f574ce9e620388376388ef32ca9d9845efca/packages/editor/src/browser/editor-preferences.ts#L107-L119

`on` for example is no longer a valid value, and the preference `id` was renamed to `files.autoSave`.

Reply:
Yes, you are right. I haven't updated that yet accordingly. I'll update this PR a bit later today.